pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -209,32 +180,48 @@ Secret phrase:       innocent throw harsh wild example reflect sausage leopard l
 
 > Save the secret phrase in a secure manner.
 
-## Creating the chain spec for the parachain
+We now have all the accounts we need and we can start prepare our chain!
 
-Let's create a chain spec for our parachain:
+## Setting up the Parachain
 
+For the sake of this exercise, let's create a new parachain:
 ```
-cd my-parachain
+pop new parachain my-parachain
 ```
+> The folder includes a `network.toml` file which can be ignored. This is to launch a network with the parachain included.
 
-Build your parachain for release:
 
-```
-pop build --release
-```
+### Creating the chain spec
 
-Create the chain spec:
+The chain specification holds all the information the node requires to start or sync with the parachain network.
+
+Let's create a chain spec for our parachain:
 
 ```
-pop build spec --release --id 2000 --type local --relay paseo-local --protocol-id my_parachain
+cd my-parachain
+pop build spec --profile release --id 2000 --type local --relay paseo-local --protocol-id my_parachain --chain local --genesis-state --genesis-code
+```
+```bash
+┌   Pop CLI : Generate your chain spec
+│
+◇  Name or path for the plain chain spec file:
+│  ./chain-spec.json
+│
+◇  Would you like to use local host as a bootnode ?
+│  No 
+│
+┌   Pop CLI : Building your chain spec
+│
+◇  Chain specification built successfully.
+│
+◆  Generated files:
+│  ● Plain text chain specification file generated at: ./chain-spec.json
+│  ● Raw chain specification file generated at: ./chain-spec-raw.json
+│  ● WebAssembly runtime file exported at: ./para-2000.wasm
+│  ● Genesis State file exported at: ./para-2000-genesis-state
+│  
+└  Need help? Learn more at https://learn.onpop.io
 ```
-
-This will output the following:
```

Comment:
Is not interesting to keep this?

Reply:
See the bash script saying exactly the same

Reply:
You are right, sorry